sample/db/samples/variants.rb

```diff
@@ -157,6 +157,96 @@
     option_values: [medium, black],
     sku: "SOL-WM006",
     cost_price: 17
+  },
+  {
+    product: solidus_snapback_cap,
+    option_values: [small],
```

Comment:
Now that we're adding a bunch of new variants, it'd be suitable to leverage automation to create all the possible combinations. However, you're building on top of what we already had, so feel free to ignore this suggestion if you think it deserves a separate PR.

Reply:
would personally prefer to refactor it in a separate PR